tools/wptrunner/wptrunner/testdriver-extra.js

```diff
@@ -160,8 +167,50 @@
         return pending_promise;
     };
 
+    /**
+     * Create an action in a specific context and return a promise that resolves when the action is complete. This is
+     * required for WebDriver Classic actions, as they require a specific context.
+     * @param name: The name of the action to create.
+     * @param context: The context in which to run the action. `null` for the current window.
+     * @param props: The properties to pass to the action.
+     * @return {Promise<any>}: A promise that resolves with the action result when the action is complete.
+     */
+    const create_context_action = function (name, context, props) {
```

Comment:
These changes were a bit tricky to understand because of naming. Actions here aren't just https://w3c.github.io/webdriver/#actions, it's any WebDriver endpoint or command. Classic endpoints now use `create_context_action` while BiDi commands use `create_action`.

No changes needed, just noting down my understanding.